### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by RiesBen**
Improving the docs:
- [x] Adding Tutorial (including filter func tutorial)
- [x] Install
- [x] index Welcome
- [x] API Fixes
- [x] Theory sections

### CODE REVIEW

File: docs/tutorial/visualization_widget.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+------------------------
+
+If you want to quickly investigate many mappings in 3D, we recommend using
+our Jupyter Notebook widget, which comes with a slider and an interactive 3D
+visualization::
+
+    from kartograf.utils.mapping_visualization_widget import display_mappings_3d
+
+    # here a list of three mappings
+    mappings: List[AtomMappings] = [mappingA, mappingB, mappingZ] #pseudo code
```
Comment (richardjgowers) [inline docs/tutorial/visualization_widget.rst:11]: could we build in some dummy mappings into the package that would help with demonstrations like this?

`from kartograf.examples import demo_mappings` or something?

File: docs/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+=========
+Tutorial
+=========
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Tutorial:
```
Comment (richardjgowers) [inline docs/tutorial.rst:7]: this caption ends up with the word Tutorial twice for no reason.  Either use the caption to say something new or remove it?

File: docs/install.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+For Developers, we recommend to setup a Kartograf environment like in the
+following example, allowing modification of the code live in the package::
+
+    git clone https://github.com/OpenFreeEnergy/kartograf.git
+
+    cd kartograf
+    mamba env create -f environment.yml
+
+    mamba activate kartograf
+    pip install -e .
```
Comment (richardjgowers) [inline docs/install.rst:32]: isn't this technically safest as `python -m pip install -e .` or something?  It was something to do with finding other versions of `pip` from root PATH

**Review by richardjgowers — COMMENTED** [review]
looking good @RiesBen !

File: docs/tutorial/comparing_mappings.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Kartograf and Lomap, you can of course always compare the number of mapped atoms::
+
+    nA = len(mappingA.componentA_to_componentB)
+    nB = len(mappingB.componentA_to_componentB)
+    nA>nB
+
+This will give you insights how many atoms get mapped by your mapping
+approaches. But this will not tell you how similar actually the mapped atoms
+are. For that we added the Jaccard Score (jcs) to the Kartograf repository,
+allowing to calculate the atom pair diversity of the mappings::
```
[the remark this answers is not in the document]
Comment (RiesBen) [reply docs/tutorial/comparing_mappings.rst:13]: s_{jaccard} = |A \cap B| / |A \cup B|

File: docs/tutorial/visualization_widget.rst (hunk 1 above)
Comment (RiesBen) [reply docs/tutorial/visualization_widget.rst:11]: that is a good Idea!

Comment (RiesBen) [reply docs/tutorial/visualization_widget.rst:11]: will add this one to the project board.

**Pull request merged**